packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx

```diff
@@ -274,10 +275,18 @@ const useGridPlans = ( {
 		const storageAddOnsForPlan =
 			isBusinessPlan( planSlug ) || isEcommercePlan( planSlug ) ? storageAddOns : null;
 
+		let isVisible = planSlugsForIntent.includes( planSlug );
+
+		// 2024-04-02 Disable upgrade to P2+
```

Comment:
Hi @jgcaruso! Thanks for taking care of sunsetting the P2+ plan. 

My intuition is telling me that it's probably more appropriate to remove the `PLAN_P2_PLUS` slug from `planSlugsForIntent` when we're first generating the `planSlugsForIntent` list.

https://github.com/Automattic/wp-calypso/blob/acb20408b2923ea80aaf0511a9aa910fb3b5d950/packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx#L199-L209

If we look into `usePlanTypesWithIntent`, well see the code for P2 plans below:

https://github.com/Automattic/wp-calypso/blob/acb20408b2923ea80aaf0511a9aa910fb3b5d950/packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx#L134-L136

Reply:
Thanks, I will give that a try. I'm definitely outside my element here, so I appreciate the advice!.

I think I initially tried to edit it there, but it wasn't working. Now that I think of it, its probably because it is in the `plans-grid-next` package and I didn't figure out I needed to re-build that package until much later in my experimenting.

Reply:
This worked, I made the change and checked it against the existing `currentSitePlanType` variable.